### ISSUE TITLE
Issue with usage via cra --template typescript

### ISSUE THREAD

**Issue by ArthurIvanov** [gregberge/xstyled#267]
Hi, I was trying to use xStyled via CRA --template typescript.
After starting dev server I get this error:

![Screen Shot 2021-05-29 at 3 47 03 PM](https://user-images.githubusercontent.com/24766420/120072638-277b8a80-c095-11eb-9fb4-2dac41d9b0d6.png)

```bash
./node_modules/@xstyled/core/dist/index.mjs
Can't import the named export 'Fragment' from non EcmaScript module (only default export is available)
```
As starting point I take your example which you have on Codesandbox.

I made repo where have this issue:
https://github.com/ArthurIvanov/cra-ts-xstyled

**Comment by theo-mesnil** [comment]
Hi @gregberge :)

On our project on NextJs we got the same issue.

**Comment by gregberge** [comment]
Hi @theo-mesnil, did you figure out a solution?

**Comment by guillaumewttj** [comment]
Hi @gregberge,

We found a solution by adding support of .mjs files as ES modules to our next.js config.
Here is the config sample:

```
  webpack(config) {
    config.module.rules.push({
      type: 'javascript/auto',
      test: /\.mjs$/,
      use: []
    })
    return config
  }
```

**Comment by agriffis** [comment]
Is this the same as #264 and possibly fixed by #275?

**Comment by gregberge** [comment]
Fixed in #275
